org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "opent2t-translator-wink-thermostat",
+  "version": "1.0.0",
+  "description": "Wink Thermostat translator",
+  "author": "OpenT2T",
+  "license": "MIT",
+  "scripts": {
+    "test": "ava"
+  },
+      "ava": {
+      "files": [
+        "tests/*.js",
+        "!**/deviceconfig.js"
+      ]
+    },
+  "dependencies": {
+    "delay": "^1.3.1",
+    "opent2t": "*",
```

Comment:
Sure. But even OpenT2T module is used only in testing,. 

---
In reply to: [73780663](https://github.com/openT2T/translators/pull/37#discussion_r73780663) [](ancestors = 73780663,73765884)